### ISSUE TITLE
Only downloads the low resolution image.

### ISSUE THREAD

**Issue by ZaibanAli** [ohyicong/Google-Image-Scraper#14]
This python script downloads images with low resolution i.e. images reduced by google (instead of full resolution).

Is there any way of downloading the original size image?

[Example here](https://prnt.sc/26q9d0r)

**Comment by ohyicong** [comment]
The code scrap that link?

**Comment by ZaibanAli** [comment]
The problem was related to Issue #15. I think google blocked my requests because I didn't use delay.

Because of that, I was getting encrypted links as a response which were compressed versions of the original images.

**Comment by rocketsfallonrocketfalls** [comment]
Hey @ZaibanAli, 

Could you please elaborate on what do you mean by that you did not use delay? Did not use which delay and where? 

I am getting the same error and I could not find a solution. Every time after 17 or 18 images downloaded, I start to get encrypted images that point to the thumbnail (preview versions) of the images instead of the actual images. As a result, all images downsized into super small images. 

Could you find a solution? 

Thanks

**Comment by ZaibanAli** [comment]
@rocketsfallonrocketfalls the delays included in GoogleImageScapper.py using time.sleep() function.

Upon further research, I found out that it was not related to delay but due to how google is opening the images.

Normally, the encrypted links are extracted only for certain images but not for all the images. If you let the script run for maybe 100 images then maybe 40 images will be with encrypted links.

I made some changes to this repo as well as one other repo to streamline the process and merged it into my own repo.  

I will upload the link to it later today.
